src/amp-story-ad-preview.js

```diff
@@ -76,6 +76,20 @@ export default class AmpStoryAdPreview {
     // TODO: Either:
     // a) purifyHtml() from ampproject/src/purifier
     // b) reject when invalid
+    dirty = dirty.replace(
+      `<head>`,
+      `<head>
+    <script>
```

Comment:
Set the script content to a constant outside the class and just interpolate here:

```js
`<head><script>${httpsCircumventionPatch}</script>`
```